ONSdigital/respondent-home-ui
# Sus021 privacy page update

# Motivation and Context
Create privacy page for social.

# What has changed
- New privacy page created.
- Pointed footer link to new page.
- Added logo link back to RH.

# How to test?
Check links work and content is correct (see: https://docs.google.com/viewerng/viewer?url=https://trello-attachments.s3.amazonaws.com/5a159c97a25f12ca639acf71/5b9a4c99a879363966d9d7a3/de6c76a030675e74ca318c23fb00bb45/Cookies_and_Privacy_(Social)_SBL_1.docx)

# Links
https://trello.com/c/d0U2sz8L/278-sus021-update-to-content-of-surveys-privacy-page-to-include-information-for-social-surveys
